ui/components/ui/textarea/index.scss

```
@@ -0,0 +1,27 @@
+@use "design-system";
+
+.textarea {
+  display: block;
+  box-shadow: none;
+  color: design-system.$black;
+
+  // not sure why this mixin isn't working: @include H6;
+  font-size: 1rem;
+  line-height: 140%;
+  font-style: normal;
+  font-weight: normal;
```

Ok so if `@use` is defined I think it overrides the usual imports and you need to traverse `design-system`

```suggestion
  @include design-system.H6;
```